dll/win32/kernel32/kernel32_vista/GetFileInformationByHandleEx.c

```diff
@@ -4,51 +4,83 @@
 #include <ndk/rtlfuncs.h>
 #include <ndk/iofuncs.h>
 
+#undef FIXME
+#define FIXME(...) do { DbgPrint(__VA_ARGS__); } while (0)
```

Comment:
Since kernel32_vista doesn't use our kernel32 debug channels,
could you
```c
#define NDEBUG
#include <debug.h>
```
and redefine FIXME with the DPRINT1 macro? So that there would be file/line info in the debug log when the FIXME is hit, instead of just four "random"-looking values being shown (from the `FIXME("%p, %u, %p, %lu\n", handle, class, info, size);` below).